sparsity/dask/core.py

```diff
@@ -135,6 +135,22 @@ def index(self):
     def map_partitions(self, func, meta, *args, **kwargs):
         return map_partitions(func, self, meta, *args, **kwargs)
 
+    # noinspection PyTypeChecker
+    def todense(self):
```

Comment:
I'd rather use to delayed here, then apply to_dense on each delayed object and use `dd.from_delayed` to construct the dense dask collection.